docs/rules/object-curly-spacing.md

```diff
@@ -37,6 +37,8 @@ Object option:
 * `"arraysInObjects": false` disallows spacing inside of braces of objects beginning and/or ending with an array element (applies when the first option is set to `always`)
 * `"objectsInObjects": true` requires spacing inside of braces of objects beginning and/or ending with an object element (applies when the first option is set to `never`)
 * `"objectsInObjects": false` disallows spacing inside of braces of objects beginning and/or ending with an object element (applies when the first option is set to `always`)
+* `"spaceInEmptyObject": "always"`: requires a space inside an empty object literal (`{ }` is valid, `{}` is invalid)
+* `"spaceInEmptyObject": "never"`: disallows a space inside an empty object literal (`{}` is valid, `{ }` is invalid)
```

Comment:
For consistency with the other options, can we change the name and schema to `emptyObjects: true` or `emptyObjects: false`?

Reply:
I'm going for consistency with the main option.

Reply:
Thanks for explaining. It looks like there is already some inconsistency here (between `always`/`never` for the main option, and `true`/`false` for the object options). Having said that, I still think it would be good to maintain consistency *among* the object options, so that we don't end up with one object option that arbitrarily has different semantics from the other options. To that end, I still think it would be better to use `true`/`false` here, and to give it a plural name without the word `space` (such as `emptyObjects`).

Reply:
I can see the argument for consistency, but personally I think always/never is more clear. True/false could be read as either, enforce space/enforce no space (i.e., equivalent to always/never), or it could be read as enforce in this case/don't enforce in this case (i.e., false = ignore, and true would mean always or never specifically).

I'm willing to bow towards consistency, but I think we should seriously reconsider the use of boolean options when the false could be read to mean either enforcing a lack of something, or not enforcing something.

Reply:
I would also be fine with changing all of the options to accept `always`/`never` (and deprecating the boolean versions). I just don't want to enforce one convention for one option, and a different convention for another option.

Reply:
Before making this change, I want to make sure we're clear.

These values will remain for backward compatibility. At least until the next major release, maybe longer.

> * `"objectsInObjects": true` requires spacing inside of braces of objects beginning and/or ending with an object element (applies when the first option is set to `never`)
 > * `"objectsInObjects": false` disallows spacing inside of braces of objects beginning and/or ending with an object element (applies when the first option is set to `always`)

These new values will be added. Note the big difference is "objectsInObjects" et. al. are now decoupled from the first option. 

 > * `"objectsInObjects": "always"` requires spacing inside of braces of objects beginning and/or ending with an object element 
 > * `"objectsInObjects": "never"` disallows spacing inside of braces of objects beginning and/or ending with an object element 

Should I bring this up in the issue thread? I'd like to have consensus before moving forward.

Reply:
If we're going to allow `always`/`never` as aliases for `true`/`false` for all the options, I think we might want to make that a separate issue and PR so that we don't block adding this option. My preference would be to use `true`/`false` with `emptyObjects` for now so that we can remain consistent, and then consider allowing `always`/`never`  for all the options separately.